src/styles/index.scss

```diff
@@ -1,3 +1,3 @@
-body {
-  margin: 0;
-}
+@import "reset-css";
```

Comment:
Consider using a more specific file name for the reset css import, such as '_reset.scss'.

Reply:
isn't fixed